packages/fxa-settings/src/lib/auth-errors/auth-errors.test.ts

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { composeAuthUiErrorTranslationID, AuthUiErrorNos } from './auth-errors';
+
+
+ describe('composeAuthUIErrorTranslationID', () => {
+   it('returns an empty string if passed no error number', () => {
+    let errorWithNoErrorNumber = { message: "I'm an error with no error number!" };
+    expect(
+      composeAuthUiErrorTranslationID(errorWithNoErrorNumber)
+    ).toEqual('');
```

Comment:
Woo thanks for the test coverage Sorry to be the test prude here but can we also add two tests (or just add two `expects` in existing tests for each sentry report) for the Sentry logging? I think you just have to mock the module out, we really should have a test present in `AppErrorBoundary` tests that I was going to refer you to but we don't. You can probably refer to payments or you could just mock it like we do metrics calls.

Reply:
test prude solidarity forever 🤝 !!! That's a great idea, sorry I just got too intimidated when I tried to set it up